pymc3/tests/test_distributions.py

```diff
@@ -2933,6 +2942,48 @@ def test_car_logp(size):
     assert np.allclose(delta_logp - delta_logp[0], 0.0)
 
 
+@pytest.mark.parametrize("sparse", [True, False])
+def test_car_rng_fn(sparse):
+    delta = 0.05  # limit for KS p-value
+    n_fails = 100  # Allows the KS fails a certain number of times
```

Comment:
Why such a large `n_fails`?

Reply:
Becouse I forgot to change it back after experimenting with it. Thank you, I fixed it.